### ISSUE TITLE
Lambdatest Identifies color-contrast ratio - specific elements but axe-core extension does not

### ISSUE THREAD

**Issue by Shivam93937** [dequelabs/axe-core#4825]
### Product

axe Extension

### Product Version

4.110.0

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation
I expected the results of axe-core extension to cover all audits provided by lambdatest on full page scan.
#### Actual
It misses out certain elements which does not have sufficient contrast ratio.

#### How to Reproduce
play.geforcenow.com

#### Additional context
Run lambdatest extension and axe-core extension (full page scan on both tools), and observe.
Axe-core does not report elements with insufficient contrast ratio that lambdatest does.

<img width="3840" height="1967" alt="Image" src="https://github.com/user-attachments/assets/f8fdb6cb-c5dc-4254-bc5b-b6cd37cd0bc8" />
<img width="3840" height="1427" alt="Image" src="https://github.com/user-attachments/assets/6b62ab7d-8124-4f4a-bb78-9b6e8b81bf69" />

**Comment by WilcoFiers** [comment]
@Shivam93937 We'd need a page to reproduce the issue on to be sure.


From the look of it though, you have "Needs review" turned off in the axe DevTools Extension, which is why you're seeing it in lambdatest and not with axe. The axe extension prompts you if you want to have it displayed after your first scan. You can adjust this in the axe extension settings.

**Comment by Shivam93937** [comment]
The page is -> play.geforcenow.com
"Needs review" option appears to be for pro members only?

**Comment by straker** [comment]
Closing due to inactivity, and Wilco's response. Please feel free to open a new bug if the problem is unresolved.
